src/actors/client/ActorClient/ActorClientGRPC.ts

```diff
@@ -0,0 +1,278 @@
+    const transactionItems: TransactionalActorStateOperation[] = [];
+
+    for (const o of operations) {
+      const transactionItem = new TransactionalActorStateOperation();
+      transactionItem.setKey(o.request.key);
+      transactionItem.setOperationtype(o.operation);
+
+      const msgSerialized = new Any();
+      msgSerialized.setValue(Buffer.from(`${o.request.value}`, "utf-8"));
+      transactionItem.setValue(msgSerialized);
+
+      transactionItems.push(transactionItem);
+    }
+
+    const msgService = new ExecuteActorStateTransactionRequest();
+    msgService.setActorType(actorType);
+    msgService.setActorId(actorId);
+    msgService.setOperationsList(transactionItems);
+
+    return new Promise((resolve, reject) => {
+      const client = this.client.getClient();
+      client.executeActorStateTransaction(msgService, (err, res) => {
+        if (err) {
+          return reject(err);
+        }
+
+        // https://docs.dapr.io/reference/api/state_api/#request-body-1
+        return resolve();
+      });
+    });
+  }
+
+  async stateGet(actorType: string, actorId: string, key: string): Promise<KeyValueType | string> {
+    const msgService = new GetActorStateRequest();
+    msgService.setActorType(actorType);
+    msgService.setActorId(actorId)
+    msgService.setKey(key);
+
+    return new Promise((resolve, reject) => {
+      const client = this.client.getClient();
+      client.getActorState(msgService, (err, res: GetActorStateResponse) => {
+        if (err) {
+          return reject(err);
+        }
+
+        // https://docs.dapr.io/reference/api/actors_api/#http-response-codes-2
+        const resData = Buffer.from(res.getData()).toString();
+
+        try {
+          const json = JSON.parse(resData);
+          return resolve(json);
+        } catch (e) {
+          return resolve(resData);
+        }
+      });
+    });
+  }
+
+  async reminderCreate(actorType: string, actorId: string, name: string, reminder: ActorReminderType): Promise<void> {
+    const msgService = new RegisterActorReminderRequest();
+    msgService.setActorType(actorType);
+    msgService.setActorId(actorId);
+    msgService.setName(name);
+
+    if (reminder.data) {
+      msgService.setData(Buffer.from(reminder?.data.toString(), "utf-8"))
+    }
+
+    if (reminder.period) {
+      msgService.setPeriod(reminder.period.toString());
+    }
+
+    if (reminder.dueTime) {
+      msgService.setDueTime(reminder.dueTime.toString());
+    }
+
+    return new Promise((resolve, reject) => {
+      const client = this.client.getClient();
+      client.registerActorReminder(msgService, (err, res) => {
+        if (err) {
+          return reject(err);
+        }
+
+        // https://docs.dapr.io/reference/api/actors_api/#http-response-codes-3
+        return resolve();
+      });
+    });
+  }
+
+  // @todo: not implemented, cannot find the gRPC bindings
+  // async reminderGet(actorType: string, actorId: string, name: string): Promise<void> {
```

Comment:
There is no reason to implement this. Please, remove the code. Once registered, reminders are kept in the runtime.